sound/soc/sof/ipc4-priv.h

```diff
@@ -73,6 +73,7 @@ struct sof_ipc4_fw_library {
  * @max_libs_count: Maximum number of libraries support by the FW including the
  *		    base firmware
  * @fw_context_save: Firmware supports full context save and restore
+ * @libraries_restored: The libraries have been retained during firmware boot
```

Comment:
bit: you mean restored during firmware boot?

Reply:
yes, just wanted to shuffle the terms a bit, should I use restored here as well?